06_PRC_Handson_User-Training.md

```diff
@@ -90,141 +98,172 @@ with iRODSSession(irods_env_file=env_file) as session:
 ```
 
-For the python object having the `__dict__` attribute, you can use the builtin `vars()` function to see many useful information about the object you instantiated.
+For any python object having a `__dict__` attribute, you can use the builtin `vars()` function to see useful information about the object.
 
 ```py
->>> b=session.data_objects.get("/yourZone/home/userName/test1/test.txt", "/yourLocalPath/test.txt")
+>>> b = session.data_objects.get("/yourZone/home/userName/test1/test.txt", "/yourLocalPath/test.txt")
 >>> vars(b)
-{'manager': <irods.manager.data_object_manager.DataObjectManager object at 0x7f534659ef10>, 'collection': <iRODSCollection 10183 b'test1'>, 'id': 10198, 'collection_id': 10183, 'name': 'test.txt', 'replica_number': 0, 'version': None, 'type': 'generic', 'size': 25, 'resource_name': 'netapp', 'path': '/yourZone/home/userName/test1/test.txt', 'owner_name': 'userName', 'owner_zone': 'yourZone', 'replica_status': '1', 'status': None, 'checksum': None, 'expiry': '00000000000', 'map_id': 0, 'comments': None, 'create_time': datetime.datetime(2021, 10, 20, 20, 39, 45), 'modify_time': datetime.datetime(2021, 10, 20, 20, 59), 'resc_hier': 'default;netapp', 'resc_id': '10014', 'replicas': [<irods.data_object.iRODSReplica netapp>], '_meta': None}
+{'manager': <irods.manager.data_object_manager.DataObjectManager object at 0x7f534659ef10>,
+'collection': <iRODSCollection 10183 b'test1'>,
+'id': 10198,
+'collection_id': 10183,
+'name': 'test.txt',
+'replica_number': 0,
+'version': None,
+'type': 'generic',
+'size': 25,
+'resource_name': 'netapp',
+'path': '/yourZone/home/userName/test1/test.txt',
+'owner_name': 'userName',
+'owner_zone': 'yourZone',
+'replica_status': '1',
+'status': None,
+'checksum': None,
+'expiry': '00000000000',
+'map_id': 0,
+'comments': None,
+'create_time': datetime.datetime(2021, 10, 20, 20, 39, 45),
+'modify_time': datetime.datetime(2021, 10, 20, 20, 59),
+'resc_hier': 'default;netapp',
+'resc_id': '10014',
+'replicas': [<irods.data_object.iRODSReplica netapp>],
+'_meta': None}
 ```
 
 ### Getting and Setting Permissions
 
-The PRC make it possible to work with ACLs (Access Control Lists). You can list given permissions on a collection or on a data object. Also, adding a new ACL and modifying an existing one on a collection/data object in iRODS via the PRC is possible.
+<!-- Maybe this should be in a common section or at the end of the "Working with collections" section, since it applies to both types of items -->
+
+Via `session.permissions` and the `iRODSAccess` class, the PRC makes it possible to work with ACLs (Access Control Lists). You can list given permissions on a collection or on a data object as well as adding, modifying and removing ACLs.
 
 Let's now create a new collection:
 
 ```py
->>> coll = session.collections.create("/yourZone/home/userName/permission")
->>> coll.path
+>>> perm = session.collections.create("/yourZone/home/userName/permission")
```

Comment:
I think we should avoid creating a collection with 'perm' as variable name, that seems confusing.
Might be best to also go for a different name than 'permission' for the collection (this was already the case before your changes).

Reply:
A new name, like coll_permissions sounds good for now.

In the long run, we should think whether we should
- reuse a collection throughout the example 
OR
- overwrite the coll variable
OR
- create a new collection and corresponding variable every example

But we should do so consistently.